### PR TITLE
Transitions actions from Ralph 2.0

### PR DESCRIPTION
**Pull request by szok**
Copy transitions from old Ralph 2.0
- assign user
- assign owner
- unassign user
- unassign owner
- assign loan end date
- assign warehouse
- unassign licences
- change hostname

Connected to #1694

### CODE REVIEW

**Comment by ar4s** [comment]
:ok:

**Comment by ghost** [comment]
:ok:

File: src/ralph/back_office/models.py [hunk 1]
```diff
@@ -41,7 +47,92 @@ class Meta:
 
     @property
     def country_code(self):
-        return 'PL'
+        if self.owner:
+            iso2 = Country.name_from_id(int(self.owner.country)).upper()
+            return iso2_to_iso3(iso2)
+        return 'POL'
```
Comment (mkurek) [inline src/ralph/back_office/models.py:53]: i know it's already merged but:
- why country code couldn't be null/None? 
- what if owner doesn't have country specified? you'll probably get some ValueError here (IMO we should return None then)

File: src/ralph/back_office/models.py [hunk 2]
```diff
@@ -41,7 +47,92 @@ class Meta:
...
 
     def __str__(self):
         return 'BackOfficeAsset: {}'.format(self.hostname)
+
+    @transition_action
+    def assign_user(self, **kwargs):
+        self.user = get_user_model().objects.get(pk=int(kwargs['user']))
```
Comment (mkurek) [inline src/ralph/back_office/models.py:60]: it should be enough to only assign user id here (we don't need to get user instance here and db will validate foreign keys on it's own)

File: src/ralph/back_office/models.py (hunk 1 above)
Comment (szok) [inline src/ralph/back_office/models.py:53]: But country field is a required field, so there's always some country will be assigned.
Default country is: pl

**Pull request merged**